maasglobal-json-schema-validator/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maasglobal-json-schema-validator",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

Comment:
Github does not yet have CD for this package, so you'd need to wither add it to CD, or deploy it manually after the PR is merged.